openelis-openmrs/src/main/java/com/ozonehis/eip/openmrs/openelis/config/OpenmrsOpenelisConfig.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.openmrs.openelis.config;
```

Comment:
Is this the package name?
The project is **eip-openelis-openmrs** (I don't know what is the agreed naming convention here)

Reply:
I think this should be fine.
WDYT @mks-d

Reply:
Let me refactor. Thanks for pointing out